operations/iuf/workflows/managed_rollout.md

```diff
@@ -81,45 +81,59 @@ Refer to the "Configure `OpenSearch`" section in the _HPE Cray EX System Monitor
 
 1. Invoke `iuf run` with `-r` to execute the [managed-nodes-rollout](../stages/managed_nodes_rollout.md) stage on all
    nodes. This will stage data to BOS and allow the workload manager to reboot the nodes when it is ready to do so. The
-   workload manager must be configured to tell BOS to reboot nodes using this staged data. The following assumes Slurm
-   is the workload manager. Follow the instructions in the
-   section [Using staged sessions with Slurm](../../boot_orchestration/Rolling_Upgrades.md#using-staged-sessions-with-slurm)
-   of the [Rolling Upgrades using BOS](../../boot_orchestration/Rolling_Upgrades.md) documentation. These instructions
-   describe two parameters that must be set in the `slurm.conf` file. Return to these instructions after setting them.
-   If an immediate reboot of compute nodes is desired instead, add `-mrs reboot` to the `iuf run` command.
+   workload manager must be configured to tell BOS to reboot nodes using this staged data.
 
-   (`ncn-m001#`) Execute the `managed-nodes-rollout` stage.
+   - If PBS is the workload manager:
 
-    ```bash
-    iuf -a "${ACTIVITY_NAME}" run -r managed-nodes-rollout
-    ```
+     1. Create a maintenance reservation in PBS. For more information on maintenance reservations, see the
+        [_PBS Professional Administrator's Guide_](https://community.altair.com/community?id=altair_product_documentation&spa=1&filter=language%3Denglish%5Eproduct%3D20069018db0348102af07608f4961995%5Eguide_type%3DAdministration&p=1&d=asc).
 
-   **NOTE:** If the `-mrs reboot` option is used, skip the following step.
+     1. (`ncn-m001#`) Execute the `managed-nodes-rollout` stage with the `-mrs reboot` option to immediately reboot all compute nodes.
```

Comment:
Might want to add "After the reservation has started", since reservations can be scheduled to start at a certain time. We don't want to reboot compute nodes until the reservation has begun.

Reply:
Updated in next commit.